app/routeConfig.js

```diff
@@ -1,9 +1,9 @@
 import NotFound from '@containers/NotFoundPage/Loadable';
-import HomeContainer from '@containers/HomeContainer/Loadable';
 import routeConstants from '@utils/routeConstants';
+import SearchContainer from '@containers/SearchContainer/Loadable';
 export const routeConfig = {
   repos: {
```

Comment:
let's create another route for this. 

So repos will still point to HomeContainer, and we'll have some tracks that points to the SearchContainer